ApplicationLibCode/ProjectDataModel/WellLog/RimWellLogCurveCommonDataSource.cpp

```diff
@@ -1124,3 +1130,21 @@ RifReaderRftInterface* RimWellLogCurveCommonDataSource::rftReader()
 
     return nullptr;
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimWellLogCurveCommonDataSource::selectSimWell( QString wellName )
```

Comment:
For a well log curve, it is probably more relevant to select a well path based on well name in addition to a simulation well.

See `RimWellLogCurveCommonDataSource::applyNextWell()` for example stepping both simwell and well path. This can be done in a later PR.

Reply:
See `RiaWellNameComparer` for sim well and well path name tools